app/serializers/blog_serializer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class BlogSerializer
+class BlogSerializer < ApplicationSerializer
   include FastJsonapi::ObjectSerializer
```

Comment:
If `include FastJsonapi::ObjectSerializer` is in the super class, it doesn't need to be in all of the subclasses.